src/Blocks/BlockPressurePlate.h

```diff
@@ -28,7 +28,8 @@ class cBlockPressurePlateHandler :
 			return false;
 		}
 
-		return (cBlockInfo::FullyOccupiesVoxel(a_Chunk.GetBlock(a_RelX, a_RelY - 1, a_RelZ)));
+		return (cBlockInfo::FullyOccupiesVoxel(a_Chunk.GetBlock(a_RelX, a_RelY - 1, a_RelZ))  ||
+				cBlockInfo::FullyOccupiesTopVoxel(a_Chunk.GetBlock(a_RelX, a_RelY - 1, a_RelZ)) );
```

Comment:
I would suggest that to instead create a new Blocks/BlockFence.h class and put an IsBlockFence() function there and call that. It might be a bit extra work though.